frontend/src/components/Home.test.tsx

```diff
@@ -0,0 +1,38 @@
+import "@testing-library/jest-dom";
+import { render, screen, fireEvent } from "@testing-library/react";
+import { vi } from "vitest";
+
+import Home from "./Home";
+
+describe("Home", () => {
+  test("renders a heading", () => {
```

Comment:
🐘 Suggested test format:

```suggestion
describe("Given the Home page", () => {
  describe("When the page is rendered", () => {
    it("Then it displays the heading", () => {
```